lineapy/utils/config.py

```diff
@@ -60,6 +61,7 @@ def __init__(
         self.do_not_track = do_not_track
         self.logging_level = logging_level
         self.logging_file = logging_file
+        self.runtime = runtime
```

Comment:
not sure if i have context on this one but runtime shouldnt be something you can configure i think? even if you do, i think it would make sense to append this as a prefix instead of overriding what we get from the _runtime() function above.

Reply:
+1, runtime is not config.

Reply:
Makes sense; undid changes in `config.py`.